examples/http-tokio/README.md

```diff
@@ -0,0 +1,28 @@
+# Simple HTTP Server using 'tokio'
```

Comment:
Update the `README.md` file to feature similar contents to the `README.md` file from PR #97 .

Reply:
See also the `README.md` contents from other `http-...` examples.